detect.go

```diff
@@ -117,11 +173,13 @@ func Detect(buildpackYMLParser BuildpackConfigParser, configParser ConfigParser,
 			return packit.DetectResult{}, err
 		}
 
-		if config.Path == "" && projectFile == "" {
+		if runtimeConfig.Path == "" && projectFile == "" {
 			return packit.DetectResult{}, packit.Fail.WithMessage("no *.runtimeconfig.json or project file found")
 		}
 
 		if projectFile != "" {
+			logger.Debug.Subprocess("Detected '%s'", projectFile)
```

Comment:
I also like this pattern of debug logging what the buildpack has detected on. I'm drafting an RFC to set some standards for logging output and definitely think we should add this.